### PR TITLE
FM-3333: tuning up mysql readme

### PR DESCRIPTION
**Pull request by jbondpdx**
Docs cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,106 +3,108 @@
...
-If you want a server installed with the default options you can run
 `include '::mysql::server'`. 
 
-If you need to customize options, such as the root
-password or `/etc/my.cnf` settings, then you must also pass in an override hash:
+To customize options, such as the root
+password or `/etc/my.cnf` settings, you must also pass in an override hash:
 
-~~~
+```
```
Comment (hunner) [inline README.md]: Should these be ```puppet?

Comment (bmjen) [inline README.md]: ^ Yeah. In the more recent README rewrites, I've seen them as ```puppet.

**Comment by jbondpdx** [comment]
Now with moar puppets!

File: README.md [hunk 2]
```diff
@@ -132,40 +134,37 @@ mysql::db { 'mydb':
...
-users on an existing MySQL server. For this to work, you need an
-appropriate `.my.cnf` in `root`'s home directory containing the remote
-server address and credentials. For example:
+To instantiate databases and users on an existing MySQL server, you need a `.my.cnf` file in `root`'s home directory. This file must specify the remote server address and credentials. For example:
 
-    [client]
-    user=root
-    host=localhost
-    password=secret
+```puppet
```
Comment (bmjen) [inline README.md]: extra puppet here.

File: README.md [hunk 3]
```diff
@@ -233,17 +230,17 @@ This parameter no longer does anything. It exists only for backwards compatibili
 
 ##### `override_options`
 
-The hash of override options to pass into MySQL. Structured like a hash in the my.cnf file:
+Specifies override options to pass into MySQL. Structured like a hash in the my.cnf file:
 
-~~~
+```
```
Comment (bmjen) [inline README.md]: needs a puppet here.

**Pull request merged**